crates/rolldown/src/stages/generate_stage/mod.rs

```diff
@@ -44,7 +43,7 @@ impl<'a> GenerateStage<'a> {
   }
 
   #[tracing::instrument(skip_all)]
-  pub async fn generate(&mut self) -> BatchedResult<Vec<Output>> {
+  pub async fn generate(&mut self) -> Result<Vec<Output>> {
```

Comment:
This `anyhow::Result` aslo eats the internal source map error, which is not expected.

Reply:
Fixed it.